packages/cli/src/utils/compiler.ts

```diff
@@ -90,4 +129,13 @@ export const compilePlatformApp = async ({ outputName, projectPath = '' }: Compi
             },
         },
     });
+
+    // move and cleanup
+    try {
```

Comment:
Can you explain why we have a `temp` folder?

Reply:
We are doing 2 builds. 1 that uses the index.ts as an entrypoint and creates a settings--XXXX.js file and another build just after that to create the app. Before every build the /dist folder is cleaned up, therefore we first build the settings specific build into a temp folder and then after the app is built move the settings-xxx.js to the /dist and cleanup. 

We have some javascript in our index.html, vite bundles this into the index.js file and because of that we are not able to import the {app, settings} export from the .js file directly and need to do a "lib" build to get to this file...

Reply:
You can prevent that with `build.emptyOutDir` ([docs](https://vitejs.dev/config/build-options.html#build-emptyoutdir))

Reply:
oh thx, ill try that out